histoqc/TileExtractionModule.py

```diff
@@ -0,0 +1,577 @@
+import os
+
+import openslide
+import json
+from histoqc.BaseImage import BaseImage
+from typing import Callable, Literal, Dict, Any, List, Tuple, get_args
```

Comment:
You import Literal and get_args from typing. However, Literal and get_args require Python >= 3.8. As I know from **readme.md** - "The HistQC tested with Python 3.7 and 3.8 Note: the DockerFile installs Python 3.8". There is error message - ImportError: cannot import name 'Literal' and 'get_args'. The users have to pip install **typing-extensions** if their Python version < 3.8. Also, need to add followed code into TileExtractionModule.py
```python
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal

try:
    from typing import get_args
except ImportError:
    from typing_extensions import get_args
```

Reply:
Thanks for pointing it out. The corresponding dynamic import was added. `requirements.txt` and the conda env yml file are updated.